CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing to forte-music/web
+
+Thanks for taking the time to contribute to forte. Here are some guidelines to follow when contributing.
+
+Some of the code in this project hasn't been refactored to follow these guidelines yet.
+
+## CSS
+
+For the most part, CSS is written using [styled-components]. Currently, they are scattered near the presentational components they are used in, in `src/components/styled` and in `src/components`. Going forward, mixins should be placed in `src/styled-mixins`, shared styled components in `src/components/styled` and unshared styled components, near where they are used.
+
+Avoid using theming unless needed. They add un-needed complexity for most use cases. If you are using the same component with slightly different styles in two different places, then use themeing.
+
+## Redux
+The [render-props] pattern is used to decouple data providing components with components consuming data. We use a wrapper around `connect` (a Higher Order Component from `react-redux`) to use the render props pattern with redux. This wrapper is called `createReduxComponent` and can be found in `src/redux/render.tsx`.
+
+## Component Folder Structure
+There are a few different types of components.
+
+* Presentational Components
+  Stateless components which are primarily concerned with how things look. State and actions are passed as props and prop callbacks respectively. These are usually functional components. Usually placed in `src/components/`.
```

Comment:
Markdown combines these two lines, so add a colon or some other separator after the first line.